tools-containers/README.md

```diff
@@ -0,0 +1,3 @@
+# DC/OS Commons Docker Images
```

Comment:
Um, the agreement with Kai was to name the directory `tools-internal` IIRC.

Reply:
So, what's the difference with the `tools` directory then? 🤔 They are also internal...

Reply:
The difference is described in https://github.com/mesosphere/dcos-commons/pull/3049#pullrequestreview-237214697

Reply:
Ok, renamed. Waiting for tests to pass and going to merge 👍